libretro/RetroArch
# [iOS/tvOS] Update bundled assets for functional initial install

## Description

Update the bundled assets.zip so that there are available overlays for use after the initial app install. This will help when App Store reviewers test the app during review.

To test:

- Do a fresh install of the app
- When the app loads, verify that the glui menu is shown and the fonts are correctly loaded
- Choose "Load Core"
- Choose "2048"
- Choose "Start Core"
- Verify that the new default RA controller overlay is shown

![Screenshot 2024-04-24 at 9 36 22 PM](https://github.com/libretro/RetroArch/assets/564774/dbdf1b9d-ba51-49bc-be7b-697a9be223b4)

## Reviewers

@warmenhoven
